html/index.html

```diff
@@ -205,111 +205,144 @@ <h2 class="my-0 fw-semibold">Nautobot Core</h2>
                             </li>
                             <li class="list-group-item"><a
-                                    href="https://docs.nautobot.com/projects/netbox-importer/en/latest/"
-                                    class="link-dark">Nautobot NetBox Importer</a><span
+                                    href="https://docs.nautobot.com/projects/circuit-maintenance/en/stable/"
+                                    class="link-dark">Nautobot Circuit Maintenance</a><span
                                     class="badge text-bg-dark rounded-pill ms-1">App</span>
                             </li>
                             <li class="list-group-item"><a
-                                    href="https://docs.nautobot.com/projects/welcome-wizard/en/latest/"
+                                    href="https://docs.nautobot.com/projects/welcome-wizard/en/stable/"
                                     class="link-dark">Nautobot Welcome Wizard</a><span
                                     class="badge text-bg-dark rounded-pill ms-1">App</span>
                             </li>
                             <li class="list-group-item"><a
-                                    href="https://docs.nautobot.com/projects/circuit-maintenance/en/latest/"
-                                    class="link-dark">Nautobot Circuit Maintenance</a><span
+                                    href="https://docs.nautobot.com/projects/capacity-metrics/en/stable/"
+                                    class="link-dark">Nautobot Capacity Metrics</a><span
                                     class="badge text-bg-dark rounded-pill ms-1">App</span>
                             </li>
+                            <li class="list-group-item"><a
+                                    href="https://docs.nautobot.com/projects/netbox-importer/en/stable/"
+                                    class="link-dark">Nautobot NetBox Importer</a><span
+                                    class="badge text-bg-dark rounded-pill ms-1">App</span>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="col">
+                    <div class="card mb-4 rounded-3 shadow-sm h-100">
+                        <div class="card-header py-3">
+                            <h2 class="my-0 fw-semibold">Commercial Software</h2>
+                        </div>
+                        <div class="card-body">
+                            <p class="card-text">This section links to documentation for commercial Nautobot Products.
+                            </p>
+                        </div>
+                        <ul class="list-group list-group-flush border-bottom-0">
+                            <li class="list-group-item"><a href="https://networktocode.com/nautobot/nautobot-cloud/"
+                                    class="w-100 btn btn-lg btn-success" target="_blank">Nautobot Commercial
```

Comment:
It's a little bit confusing that there are 2 links to Nautobot Cloud in this section but one of them is labeled "Nautobot Commercial Software"